preview/drm_preview.cpp

```diff
@@ -269,38 +269,38 @@ DrmPreview::~DrmPreview()
 	close(drmfd_);
 }
 
-void DrmPreview::makeBuffer(int fd, size_t size, unsigned int width, unsigned int height, unsigned int stride,
-							Buffer &buffer)
+void DrmPreview::makeBuffer(int fd, size_t size, StreamInfo const &info, Buffer &buffer)
 {
 	buffer.fd = fd;
 	buffer.size = size;
-	buffer.width = width;
-	buffer.height = height;
-	buffer.stride = stride;
+	buffer.width = info.width;
+	buffer.height = info.height;
+	buffer.stride = info.stride;
```

Comment:
Could put the entire info in the buffer, though again, it seems scarcely worth the trouble. TBH, looking at what's below, do we even need these values at all? Maybe a question for another time.